app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class QuestionsController < ApplicationController
+    
+    def get_questions
+
+        @Question = Question.all
```

Comment:
This shouldn't be capitalized and it should be plural as `.all` returns the list of all questions.